sbi/inference/posteriors/sbi_posterior.py

```diff
@@ -42,14 +42,15 @@ def __init__(
             algorithm_family: one of 'snpe', 'snl', 'sre' or 'aalr'
             neural_net: a classifier for sre/aalr, a density estimator for snpe/snl   
             prior: prior distribution with methods `log_prob` and `sample`
-            context: observations acting as conditioning variables. Absent if 
-                None. If provided, it must have same leading dimension as the inputs.  
+            x_o: observations acting as conditioning variables. Absent if 
+                None. If provided, it must have same leading dimension as the inputs.
+                TODO: This is unclear to me. Why is x_o optional? And what are inputs?
```

Comment:
I deleted the  `If provided, it must have same leading dimension as the inputs.` comment. I think in general it might make sense to let `x_o` be optional, as you mentioned. So I am leaving this as it is for now.